packages/vats/tools/board-utils.js

```diff
@@ -1,17 +1,17 @@
 // @ts-check
 /**
  * @typedef {{
- *   brand: import('@agoric/internal/src/marshal.js').BoardRemote & Brand,
+ *   brand: import('@agoric/internal/src/marshal.js').BoardRemote,
```

Comment:
Losing `Brand` is a bummer.

I have come to realize that the unmarshalled objects don't need to know their board id - the correspondence can be kept elsewhere. That's the approach I'm using in 2d7a2c5 in #7939.

But I suppose the current code relies on `getBoardId()`, so `BoardRemote` is the relevant type.